nh_eobs/sql_statements.py

```diff
@@ -352,16 +352,18 @@ def get_workload(self, settings):
                 else false
             end as notification
         from nh_activity activity
-        inner join nh_activity spell on spell.id = activity.parent_id
+        inner join nh_activity spell_activity
```

Comment:
Renaming this confusing join name as it's the spell activity and not the spell that this represents